app/src/main/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdapters.java

```diff
@@ -4,15 +4,18 @@
 import androidx.annotation.ColorInt;
 import androidx.annotation.NonNull;
 import androidx.appcompat.widget.AppCompatCheckBox;
+import androidx.core.widget.CompoundButtonCompat;
 import androidx.databinding.BindingAdapter;
 
 /**
  * Custom data-binding adapters for {@link AppCompatCheckBox}s.
  */
 public final class AppCompatCheckBoxBindingAdapters {
-  /** Sets the button tint for the specified checkbox, via data-binding. */
+  /**
+   * Sets the button tint for the specified checkbox, via data-binding.
+   */
```

Comment:
Single line KDocs need to be on one line, as it was before. Please revert this. You can check out the kotlin style guide that is linked in our wiki.

Reply:
My mistake. I have been looking using double shift. But forgot about ctrl +shift + f.
About code style - I'll read read more carefully this section